selfdrive/car/honda/carstate.py

```diff
@@ -30,6 +31,16 @@ def parse_gear_shifter(can_gear_shifter, car_fingerprint):
       return "sport"
     elif can_gear_shifter == 0x20:
       return "low"
+    
+  elif car_fingerprint in (CAR.PILOT):
+     if can_gear_shifter == 0x8:
+       return "reverse"
+     elif can_gear_shifter == 0x4:
+       return "park"
+     elif can_gear_shifter == 0x20:
+       return "drive"
+     elif can_gear_shifter == 0x2:
+        return "sport"
```

Comment:
Here is a bug. This function will never return "sport" because at line 14 you are assuming that for all Hondas `0x02` is "reverse"

Reply:
Good catch. Will fix this in internal repo and it will be in next release. Currently, this bug doesn't result in unexpected behaviors as both S and R don't allow OP engagement.

Reply:
Can you guys add support to "sport" mode 🙏?  Stock ACC work on sport mode.
I have this line patched to return "drive" so OP can drive in sport mode.

Or maybe just do what I'm doing and return "drive"  instead of "sport" for all other cars. If you'd like I can open a PR for this.

Reply:
@mcampa , the rational behind not allowing OP in Sport mode is that it might require a re-tuning, since the car controls might respond differently. Not sure if that's even true actually...